tests/base.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+import run
+from run import app, g
+from flask_testing import TestCase
+from database import create_session
+from mod_home.models import GeneralData, CCExtractorVersion
+
+
+class BaseTestCase(TestCase):
+
+    def create_app(self):
+        """
+        Create an instance of the app with the testing configuration
+        :return:
+        """
+        app.config['TESTING'] = True
+        app.config['DATABASE_URI'] = 'sqlite:///:memory:'
+        app.config['WTF_CSRF_ENABLED'] = False
+        app.config['SQLALCHEMY_POOL_SIZE'] = 1
+        return app
+
+    def setUp(self):
+        pass
+
+    @staticmethod
+    def add_last_commit_to_general_data():
```

Comment:
If these are called outside the Flask framework, the g.db will not be defined. I'd rather see you pass in the g.db as an argument in the calls.